Tropos/Controllers/TRAppDelegate.m

```diff
@@ -38,6 +44,35 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken;
+{
+    NSString *channel = [[NSTimeZone defaultTimeZone] name];
+    [self.courier subscribeToChannel:channel withToken:deviceToken];
+}
+
+- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(nonnull void (^)(UIBackgroundFetchResult))completionHandler;
+{
+    if (![userInfo valueForKeyPath:@"aps.content-available"]) {
```

Comment:
Do you think it's worth checking the actual value here, rather than just the presence of the key? Technically the value should be `@1`.